test/benchmarks.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint no-console: "off" */
+import { constants as fsConstants, promises as fs } from "fs";
+import path from "path";
+import { cwd } from "process";
+import { Repository, Clone } from "nodegit";
+import { ESLint } from "eslint";
+import Benchmark from "benchmark";
+
+// Explicitly exit with non-zero when there is some error
+process.on("unhandledRejection", (err) => {
+  throw new Error(err);
+});
+
+type RepoInfo = {
+  name: string; // name of repo
```

Comment:
nit: might be better to move comments to above the properties as to follow style convention